src/Kernel32.Desktop/Kernel32.cs

```diff
@@ -684,5 +684,67 @@ public static partial class Kernel32
         /// </returns>
         [DllImport(nameof(Kernel32), SetLastError = true)]
         public static extern bool IsWow64Process(SafeObjectHandle hProcess, out bool Wow64Process);
+
+        /// <summary>Retrieves the window handle used by the console associated with the calling process.</summary>
+        /// <returns>
+        ///     The return value is a handle to the window used by the console associated with the calling process or
+        ///     <see cref="IntPtr.Zero" /> if there is no such associated console.
+        /// </returns>
+        [DllImport(nameof(Kernel32))]
+        public static extern IntPtr GetConsoleWindow();
```

Comment:
Well in this scenario I would just have my generator look at MSDN (here: https://msdn.microsoft.com/en-us/library/windows/desktop/ms683175(v=vs.85).aspx) and conclude that it is a `HWND` type, and then convert that into a `IntPtr`.